geopandas/base.py

```diff
@@ -790,6 +790,34 @@ def cx(self):
         """
         return _CoordinateIndexer(self)
 
+    def equals(self, other):
+        """
+        Test whether two objects contain the same elements.
+
+        This function allows two GeoSeries or GeoDataFrames to be compared
+        against each other to see if they have the same shape and elements.
+        Missing values in the same location are considered equal. The column
+        headers do not need to have the same type (as long as they are
```

Comment:
Sorry to enter this conversation late, but what does this actually mean?
> The column headers do not need to have the same type

This also doesn't apply when comparing between `GeoSeries`, right?

Reply:
To be clear, this is a copy of pandas' docstring, so also what you would have seen before for `GeoSeries/DataFrame.equals?`. 

(that said, I certainly agree it's not the clearest docstring .. ;))

> This also doesn't apply when comparing between GeoSeries, right?

I suppose it's "labels" in general, so the index labels for Series, and the column + row labels for DataFrame